Assets/Scripts/UI/MainMenuUI.cs

```diff
@@ -5,10 +5,12 @@
 using UnityEngine.SceneManagement;
 using BTManager;
 using StateMachine;
+using Common;
+using Reward;
 
 namespace UI
 {
-    public class MainMenuUI : MonoBehaviour
+    public class MainMenuUI : Instance<MainMenuUI>
```

Comment:
Instance?

Reply:
So I can access it in the scene without reference.